.github/workflows/automatic_code_review.yml

```diff
@@ -0,0 +1,77 @@
+name: Automatic code review
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  review_code:
+    name: Run Danger 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Checkout source code
+        uses: actions/checkout@v2.3.2
+
+      - name: Load Gradle cached
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-*
+            ~/.gradle/caches/jars-*
+            ~/.gradle/caches/build-cache-*
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      # Run Detekt and Archive Detekt reports
+      - name: Run Detekt on CoroutineTemplate
+        working-directory: ./CoroutineTemplate
+        run: ./gradlew detekt
+
+      - name: Run Detekt on RxJavaTemplate
+        working-directory: ./RxJavaTemplate
+        run: ./gradlew detekt
+
+      - name: Run Android Lint on RxJavaTemplate
+        working-directory: ./RxJavaTemplate      
+        run: ./gradlew lint
+
+        # Run Unit Test & Jacoco for code coverage & Archive reports
+      - name: Run Unit Tests & Jacoco on RxJavaTemplate
+        working-directory: ./RxJavaTemplate
+        run: ./gradlew jacocoTestReport
+
+      # Setup Ruby gems and Bundle for executing danger-ruby
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+
+      - name: Cache Gems
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Bundle Install
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+
+      - name: Check Versions
+        run: |
+          ruby --version
+          gem --version
+          bundler --version
```

Comment:
maybe just add this to the above step, after installing we can `echo 'version checking'` or sth like that; it relates to the step above so running together seems ok to me